src/views/ViewTrackDetails.tsx

```diff
@@ -115,6 +133,130 @@ export default function ViewTrackDetails() {
             }}
           />
         </Setting.Section>
+        <Setting.Section>
+          <Setting.Input
+            label="Year"
+            id="year"
+            name="year"
+            type="number"
+            min="1900"
```

Comment:
You never know ;) classical music and all that. Min: 0 would probably be safer, even if highly improbable.